CHANGELOG.md

```diff
@@ -90,6 +90,7 @@ System Console
 
 - IE 11 new minimum version for IE, since IE 10 share fell below 5% on desktop 
 - Safari 8 new minimum version for Safari, since Safari 7 fell below 1% on desktop 
+  - Note: Safari 8 is temporarily disabled in v1.2 due to a CSS issue and support should return in v1.3. Safari 9 is working.
```

Comment:
I don't think we plan to bring back support at all for safari 8. I would certainly suggest against it.